cmd/serve/serve.go

```diff
@@ -102,6 +103,22 @@ var ServeCmd = &cobra.Command{
 				}
 				return int32(topK)
 			}
+			maxTokens := func() int {
+				env := os.Getenv("K8SGPT_MAX_TOKENS")
+				if env == "" {
+					return defaultMaxTokens
+				}
+				maxTokens, err := strconv.ParseInt(env, 10, 32)
+				if err != nil {
+					color.Red("Unable to convert maxTokens value: %v", err)
+					os.Exit(1)
+				}
+				if maxTokens < 10 || maxTokens > 32768 {
```

Comment:
Rather than setting the limit here, it probably works better for the specific checks inside the AI backend

Reply:
Agree, happy if I remove the range check here?

Limits for max tokens per AI can be updated in a future PR, I think this should also apply to topk as it varies per backend (I followed the same generic check approach)